configure

```diff
@@ -241,6 +241,7 @@ code=
 
 int main(void)
 {
+	crypto_secretstream_xchacha20poly1305_state crypto_state;
```

Comment:
This doesn't work, because gcc gives an unused variable warning!

I took the actual code we use in the hsm, and simplified it a little.